.github/workflows/devnet-deploys.yml

```diff
@@ -51,7 +51,6 @@ jobs:
       - uses: actions/checkout@v4
         with:
           ref: "${{ env.GIT_COMMIT }}"
-          fetch-depth: 0
```

Comment:
fetch depth 0 meant github woult checkout all branches and tags. Not sure if the workflow file needed this or not?
cc @spypsy

Reply:
It's needed for `git diff` operation in `build`, perhaps there's a better option though?
Can probably remove it from the rest of the steps

Reply:
Fixed :)